src/lib/layer-manager.js

```diff
@@ -77,7 +80,18 @@ export default class LayerManager {
         layerId: null
       }
     };
+
     Object.seal(this.context);
+
+    seer.listenFor('deck.gl', payload => {
```

Comment:
I think it would be nicer if this was a function imported from `seer-integration.js`